lona/static_file_loader.py

```diff
@@ -170,15 +166,15 @@ def discover(self):
         self.style_tags_html = self.server.templating_engine.render_template(
             self.server.settings.STATIC_FILES_STYLE_TAGS_TEMPLATE,
             {
-                'stylesheets': [i for i in self.node_stylesheets if i.enabled],
+                'stylesheets': [i for i in self.node_stylesheets if i.enabled and i.link],  # NOQA: LN001
```

Comment:
Could you put this in a separate line or add a newline?

Reply:
Fixed